### DISCUSSION TITLE
dns records aren't updated

### DISCUSSION THREAD

**Discussion by dario-pulcinelli** [freeipa/ansible-freeipa#1269] [General]
Hello team,

During the last months I've been using this role I've noticed that the ip addresses aren't updated in the DNS even specifying ipasssd_enable_dns_updates=yes. I'm re-deploying the machines, so I use ipaclient_force_join=yes as well but the ip addresses aren't being updated in the dns database. Can you please investigate ?

Thanks

**Comment by rjeffman** [comment]
Sorry for the late response, but somehow Github stop sending me emails on new issues and discussions. I'll loook into it.

**Comment by gorgonzola5000** [comment]
https://github.com/freeipa/ansible-freeipa/issues/1346
